src/ui/template.js

```diff
@@ -26,35 +28,57 @@ export default class Template {
 		 *
 		 * @property {TemplateDefinition}
 		 */
-		this.def = def;
+		this.definition = def;
 	}
 
 	/**
-	 * Renders HTMLElement using {@link #def}.
+	 * Renders DOM Node using {@link #definition}.
 	 *
 	 * @returns {HTMLElement}
 	 */
 	render() {
-		return this._renderElement( this.def, true );
+		return this._renderNode( this.definition, true );
 	}
 
 	/**
-	 * Renders an element from definition.
+	 * Renders a DOM Node from definition.
 	 *
 	 * @protected
-	 * @param {TemplateDefinition} def Definition of an element.
+	 * @param {TemplateDefinition} def Definition of a Node.
 	 * @param {Boolean} intoFragment If set, children are rendered into DocumentFragment.
-	 * @returns {HTMLElement} A rendered element.
+	 * @returns {HTMLElement} A rendered Node.
 	 */
-	_renderElement( def, intoFragment ) {
+	_renderNode( def, intoFragment ) {
 		if ( !def ) {
```

Comment:
Remove this unnecessary check. Do not check things which will fail in the next line anyway.